extensions/EXT/EGL_EXT_protected_content.txt

```diff
@@ -85,6 +85,12 @@ New Tokens
     New EGLSurface attribute name:
 
         EGL_PROTECTED_CONTENT_EXT               0x32C0
+        
+    Accepted as an <attribute> parameter of eglQuerySurface and 
```

Comment:
These could be combined rather than repeat the enum definition twice - something like

> Accepted as an attribute name in the <attrib_list> parameter of eglCreateContext, eglCreateWindow Surface, and eglCreateImageKHR; and as an <attribute> parameter of eglQuerySurface and eglQueryContext:

Reply:
Yes, better

Reply:
Done